contracts/protocol/lendingpool/LendingPool.sol

```diff
@@ -49,7 +49,7 @@ contract LendingPool is VersionedInitializable, ILendingPool, LendingPoolStorage
   using PercentageMath for uint256;
   using SafeERC20 for IERC20;
 
-  uint256 public constant LENDINGPOOL_REVISION = 0x2;
+  uint256 public constant LENDINGPOOL_REVISION = 0x3;
```

Comment:
AAVE uses `LENDINGPOOL_REVISION` for major version upgrade, [`0x3` means AAVE v3](https://etherscan.io/address/0x87870Bca3F3fD6335C3F4ce8392D69350B4fA4E2#readProxyContract).

Should we keep `0x2` to be consistent with AAVE v2 version?

Reply:
we don't need to follow their schema here since if we don't bump the revision here, we won't be able to re-initialize the contract and update the flash loan rate variable